python/ray/data/tests/test_map.py

```diff
@@ -393,7 +414,6 @@ def test_select_columns(ray_start_regular_shared):
         assert each_ds.select_columns(cols=["col1", "col2", "col2"]).schema().names == [
             "col1",
             "col2",
-            "col2",
```

Comment:
I don't think this change is right?

Reply:
Closing because the implementation and test of select_columns have changed.